redis/src/cmd.rs

```diff
@@ -303,6 +303,16 @@ impl Cmd {
         self
     }
 
+    /// Used by the RedisJSON to allow the returning of an error when serialisation fails on some commands
+    /// **This should NOT be used by anything else**
+    #[inline]
+    #[doc(hidden)]
+    #[cfg(feature = "json")]
+    pub(crate) fn arg_take<T: ToRedisArgs>(mut self, arg: T) -> Cmd {
+        arg.write_redis_args(&mut self);
+        self
+    }
+
```

Comment:
no other use for this other than this specific use case so this is an internal function hidden from the docs and only allowed in the crate